src/main/asciidoc/installation-guide/installation-guide-configuration.adoc

```diff
@@ -337,6 +337,17 @@ ds:
     port: 9300
 ----
 
+==== Sharding tags
+
+You can apply sharding on Gateway's instances either with system properties or with a configuration as below.
```

Comment:
Could you explain which one is used if system properties and configuration are set ?